packages/holocron-module-route/src/ModuleRouteUtils.jsx

```diff
@@ -17,22 +17,23 @@ import { createRoutes } from '@americanexpress/one-app-router';
 // holocron is a peer dependency
 import { loadModule } from 'holocron'; // eslint-disable-line import/no-unresolved,import/extensions
 
-export const addToRouteProps = (route, newProps) =>
-  Object.assign({}, route, { props: Object.assign({}, route.props, newProps) });
+export const addToRouteProps = (
+  route, newProps) => ({ ...route, props: { ...route.props, ...newProps } });
 
 export const passChildrenProps = (givenRoutes = [], newProps) => {
   const routes = typeof givenRoutes === 'function' ? givenRoutes(newProps.store) : givenRoutes;
-  return Array.isArray(routes) ? routes.map(route => addToRouteProps(route, newProps)) :
-    [addToRouteProps(routes, newProps)];
+  return Array.isArray(routes) ? routes.map((route) => addToRouteProps(route, newProps))
+    : [addToRouteProps(routes, newProps)];
 };
 
-export const getRouteIndex = (routes, props) =>
-  (routes && createRoutes(passChildrenProps(routes, props))[0].indexRoute);
+export const getRouteIndex = (routes, props) => (
+  routes && createRoutes(passChildrenProps(routes, props))[0].indexRoute
+);
```

Comment:
no change required, but wanted to comment (positively 😅 PRs can be rough 😓) that I like this formatting